source/server/listener_manager_impl.cc

```diff
@@ -233,7 +240,8 @@ ListenerImpl::ListenerImpl(const envoy::api::v2::Listener& config, const std::st
         filter_chain_match.application_protocols().begin(),
         filter_chain_match.application_protocols().end());
 
-    addFilterChain(server_names, filter_chain_match.transport_protocol(), application_protocols,
+    addFilterChain(filter_chain_match.destination_port().value(), destination_ips, server_names,
```

Comment:
what if destination_port is missing?

Reply:
It's `0` by default.

Omitting those fields in configuration didn't affect existing tests, and we have pretty good coverage there.

Reply:
Other places in the codebase use PROTOBUF_GET_WRAPPED_OR_DEFAULT() for this behavior. I think this is a special case because the default for this type is 0, but it feels weird to me to get thing() when has_thing()== false.